### PR TITLE
[BugFix]Show registration message on sign up

### PR DESCRIPTION
**Pull request by jysmys**
[PT link](https://www.pivotaltracker.com/story/show/172888074)

### CODE REVIEW

File: src/components/LoginForm.jsx [hunk 1]
```diff
@@ -23,11 +25,12 @@ const LoginForm = (props) => {
         history.goBack();
       }
     } catch (error) {
-      setMessage(error.response.data.errors[0]);
+      seterrorMessage(error.response.data.errors[0]);
```
Comment (erikbjoern) [inline src/components/LoginForm.jsx]: This should be camelCased to setErrorMessage actually, as well as when declared on line 11

File: src/index.js [hunk 2]
```diff
@@ -5,17 +5,23 @@ import * as serviceWorker from "./serviceWorker";
...
 import { BrowserRouter } from "react-router-dom";
-import { StripeProvider } from 'react-stripe-elements'
+import { StripeProvider } from "react-stripe-elements";
 import "./css/index.css";
+import { Provider } from "react-redux";
+import configureStore from "./state/store/configureStore";
 
 axios.defaults.baseURL = "http://localhost:3000/api";
+const store = configureStore();
+window.store = store;
```
Comment (erikbjoern) [inline src/index.js]: Should we have this line in our implementation?

File: src/index.js [hunk 3]
```diff
@@ -5,17 +5,23 @@ import * as serviceWorker from "./serviceWorker";
...
 
 ReactDOM.render(
-  <StripeProvider apiKey="pk_test_21nBNjeqdyB1Mzm2VjDPQprF00kyEKYZSK">
-    <BrowserRouter>
-      <App />
-    </BrowserRouter>
-  </StripeProvider>,
+  <Provider store={store}>
+    <StripeProvider apiKey="pk_test_21nBNjeqdyB1Mzm2VjDPQprF00kyEKYZSK">
```
Comment (faraznaeem) [inline src/index.js]: are you not using an env file for the API key?

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 3 above)
Comment (erikbjoern) [reply src/index.js]: That's the publishable key, do we need to put it in a env file?

Comment (jysmys) [reply src/index.js]: This is merged from upstream...

File: src/index.js (hunk 2 above)
Comment (jysmys) [reply src/index.js]: No!

File: src/index.js (hunk 3 above)
Comment (jysmys) [reply src/index.js]: will put in in env file

**Review by designerofthing — APPROVED** [review]

File: src/state/store/configureStore.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { createStore } from "redux";
+import rootReducer from "../reducers/rootReducers";
```
Comment (erikbjoern) [inline src/state/store/configureStore.js]: You import 'rootReducer' in singular, from 'rootReducers' in plural, does this work? You export it as 'rootReducers'

Comment (jysmys) [reply src/state/store/configureStore.js]: Fixed! Good spotted, but weird it still worked....

Comment (jysmys) [reply src/state/store/configureStore.js]: updated now!!

**Review by oliverochman — APPROVED** [review]

**Pull request merged**